pkgs/applications/misc/bazecor/default.nix

```diff
@@ -35,14 +38,18 @@ appimageTools.wrapAppImage rec {
   # to allow non-root modifications to the keyboards.
 
   extraInstallCommands = ''
-    install -m 444 -D ${src}/Bazecor.desktop -t $out/share/applications
-    substituteInPlace $out/share/applications/Bazecor.desktop \
-      --replace 'Exec=Bazecor' 'Exec=bazecor'
+    source "${makeWrapper}/nix-support/setup-hook"
```

Comment:
Can't we add makeWrapper to nativeBuildInputs?

Reply:
I could be wrong, but iirc I don't think there's a nativeBuildInputs with appimageTools. 

If it's not available, we would need to rework the PR completely to use `mkDerivation` instead and it would make more sense to use apimageTools imo.